tester_app/client/assets/stylesheets/_react-on-rails-sass-helper.scss

```diff
@@ -0,0 +1,16 @@
+// This file is used to facilitate the proper integration of Sass with Rails.
```

Comment:
Replace above comment

// This file allows the same sass code with Rails and node-sass for the Webpack Dev Server.